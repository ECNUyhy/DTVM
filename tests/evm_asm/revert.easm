PUSH32 0xFF01000000000000000000000000000000000000000000000000000000000000
PUSH1 0
MSTORE
PUSH1 2
PUSH1 0
REVERT
