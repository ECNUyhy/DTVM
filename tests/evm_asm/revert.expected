status: revert
error_code: 2
stack: []
memory: 'FF01000000000000000000000000000000000000000000000000000000000000'
storage: {}
transient_storage: {}
return: 'FF01'
events: []
